From d6363f51fe7c93e6a2ff6a7b5c08249dd3de4c85 Mon Sep 17 00:00:00 2001
From: Russ Weight <russell.h.weight@intel.com>
Date: Thu, 17 Mar 2016 13:19:45 -0700
Subject: [PATCH 2/2] BT: ldisc: REVERTME: Fix perms for lbf_ldisc.h

This change is a fixup for the creation of a common CHT quilt.
The purpose is to retore source equivalence with the m1stable
git source. This is a file permission change.

Change-Id: Id2e066a17912629e670051f0d67ab316e4780dce
Signed-off-by: Russ Weight <russell.h.weight@intel.com>
---
 include/linux/lbf_ldisc.h | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 mode change 100644 => 100755 include/linux/lbf_ldisc.h

diff --git a/include/linux/lbf_ldisc.h b/include/linux/lbf_ldisc.h
old mode 100644
new mode 100755
-- 
1.9.1

